plasma_framework/contracts/src/exits/payment/routers/PaymentInFlightExitRouter.sol

```diff
@@ -340,7 +332,7 @@ contract PaymentInFlightExitRouter is
      * @param newBondSize The new bond size
      */
     function updateStartIFEBondSize(uint128 newBondSize) public onlyFrom(framework.getMaintainer()) {
-        startIFEBond.updateBondSize(newBondSize);
+        startIFEBond.updateBondSize(newBondSize, 0);
```

Comment:
Is this function still needed and why is exit bounty size set to 0?

Reply:
Since we are not paying the bounty as a part of the IFE Bond and to use the same struct for the bond setting it to 0?

Reply:
Ok makes sense.